src/graph/entities/attribute_set.c

```diff
@@ -212,10 +212,14 @@ void AttributeSet_Set_Allow_Null
 		return;
 	}
 
-	size_t n = ATTRIBUTESET_BYTE_SIZE(_set) + sizeof(Attribute);
-	_set = rm_realloc(_set, n);
```

Comment:
So the issue with `rm_realloc` is that in-case `_set` is NULL `rm_realloc` won't zero out the allocated space ?

Reply:
Yes